cl_sii/cte/f29/parse_datos_obj.py

```diff
@@ -57,7 +57,7 @@ def _parse_sii_cte_f29_datos_obj_to_dict(
     datos_obj: SiiCteF29DatosObjType,
     schema_validator: Callable[[SiiCteF29DatosObjType], None],
     campo_deserializer: Callable[[object, str], object],
-) -> Mapping[str, object]:
+) -> dict:
```

Comment:
Instead of modifying the return type annotation of `_parse_sii_cte_f29_datos_obj_to_dict()` and making it less specific for all callers

```suggestion
) -> Mapping[str, object]:
```

we could modify the type annotations only of the calling code that is failing type checking:

```python
# Line 47:
obj_params: Mapping[str, Any] = _parse_sii_cte_f29_datos_obj_to_dict(
```